updatecli/updatecli.d/manifest.yaml

```diff
@@ -0,0 +1,165 @@
+name: Bump CAPI providers version
+
+# define scm for turtles
+scms:
+  turtles:
+    kind: github
+    spec:
+      user: turtles-bot
+      email: turtles@suse.de
```

Comment:
is this bot already exists?

Reply:
I tested this with my own fork of Turtles and using upstream CAPI repositories as source for new releases. It's not the exact same scenario but it's the closest I could test before merging. You can take a look at a sample PR `updatecli` created in my fork [here](https://github.com/salasberryfin/rancher-turtles/pull/10). I did not have to create a bot beforehand.

Reply:
Thanks @salasberryfin, it looks good to me, although PR was created using GHA bot? I thought turtles-bot would be normal GH user created by use and part of the org with right permission to push changes.

Also, I have one follow-up question: https://github.com/salasberryfin/rancher-turtles/pull/10/files#r1810287894